psycop/common/model_training_v2/config/baseline_pipeline.py

```diff
@@ -7,11 +7,15 @@
 
 
 def train_baseline_model(cfg_file: Path) -> float:
-    cfg_raw = BaselineRegistry.fill(Config().from_disk(cfg_file))
-    cfg_schema = BaselineSchema(**BaselineRegistry.resolve(cfg_raw))
+    cfg = Config().from_disk(cfg_file)
+    cfg_schema = BaselineSchema(**BaselineRegistry.resolve(cfg))
 
     cfg_schema.logger.log_config(
-        cfg_raw,
+        cfg,
+    )
+    cfg_schema.logger.warn(
+        """Config is not filled, so defaults will not be logged.
+                           Waiting for https://github.com/explosion/confection/issues/47 to be resolved.""",
```

Comment:
Gav den tidligere kode problemer? :-)

OBS her: Når vi logger sådan her får vi dict'en efter resolution. Det betyder at values'ne er selve instancen af objektet, e.g. `cfg["trainer"]: <Baseline at x03853....>`.

Reply:
@HLasse Pinger lige her.

Reply:
`.fill` var ikke glad for * args jvf dit og Kenneth issue. Vi logger præcis det samme som før, bare rolig ;)

Reply:
Ah, feberhjernen overså at det ikke var den resolvede der blev logget! Beklager 😘

Reply:
Skynd dig i seng igen 🤒 ❤️